encrypted-media-respec.html

```diff
@@ -159,6 +159,15 @@
 
     <section id="sotd">
         <p>The <code>"persistent-usage-record"</code> session type and the related <em>MediaKeySession destroyed</em> algorithm were removed since the previous version.</p>
+        <p class="note">
+          Implementers are advised to pay extra attention to sections <a href='#security'></a> and <a
+          href='#privacy'></a>, to mitigate threats and vulnerabilities from underlying Key System
+          implementations, and to provide users with adequate control over their own privacy. In particular,
+          the requirements of the privacy and security sections cannot be met without knowledge of the privacy
+          and security properties of the Key System. If a user agent chooses to support a Key System
```

Comment:
Add "implementation" after Key System. Or "Key System and its implementation(s)".